narwhals/dtypes.py

```diff
@@ -386,6 +386,26 @@ class Datetime(TemporalType):
 
     Notes:
         Adapted from [Polars implementation](https://github.com/pola-rs/polars/blob/py-1.7.1/py-polars/polars/datatypes/classes.py#L398-L457)
+
+    Examples:
+        >>> import pandas as pd
+        >>> import polars as pl
+        >>> import pyarrow as pa
+        >>> import narwhals as nw
+        >>> from datetime import datetime, timedelta
+        >>> import pytz
```

Comment:
instead of pytz, which is deprecated, can we create the Series using tz-naive data (`datetime`) and then use `.dt.replace_time_zone`?

Reply:
Sure thing, I'd get it done right away